lib/airbrake/rake/tasks.rb

```diff
@@ -100,7 +100,10 @@
 
     exit!(1) if [id, key].any?(&:nil?)
 
+    env = heroku_env['RAILS_ENV'] || ENV['RAILS_ENV']
```

Comment:
Applied this change and added a warning.

> Just curious, does a dynamic URL value for the deploy hook? E.G "some-url?#{ENV['RAILS_ENV']}"?

No, it shouldn't work because what comes after the pound sign is not going to be sent to the server.